islands/ItemsList.tsx

```diff
@@ -77,18 +77,20 @@ export default function ItemsList(
 
   return (
     <div>
-      {itemsSig.value.length
-        ? itemsSig.value.map((item, id) => {
-          return (
-            <ItemSummary
-              key={item.id}
-              item={item}
-              isVoted={itemsAreVotedSig.value[id]}
-              isSignedIn={props.isSignedIn}
-            />
-          );
-        })
-        : <EmptyItemsList />}
+      {isLoadingSig.value !== undefined && !isLoadingSig.value
+        ? itemsSig.value.length > 0
+          ? itemsSig.value.map((item, id) => {
+            return (
+              <ItemSummary
+                key={item.id}
+                item={item}
+                isVoted={itemsAreVotedSig.value[id]}
+                isSignedIn={props.isSignedIn}
+              />
+            );
+          })
+          : <EmptyItemsList />
+        : "█"}
```

Comment:
Can you make this a little more readable, please? I think we should avoid nesting ternary operators.

Reply:
Sure thing